x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/processor_form/processors/network_direction.tsx

```diff
@@ -0,0 +1,264 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FunctionComponent, useState, useEffect } from 'react';
+import { i18n } from '@kbn/i18n';
+import { isEmpty } from 'lodash';
+import { EuiButtonEmpty, EuiCode } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import { FieldsConfig, from, to } from './shared';
+import { TargetField } from './common_fields/target_field';
+import { SerializerFunc } from '../../../../../../shared_imports';
+import { IgnoreMissingField } from './common_fields/ignore_missing_field';
+import {
```

Comment:
Ohh, I've been sorting them by length of the import all this time! Feels we could be using something like [this](https://github.com/trivago/prettier-plugin-sort-imports) to automate it

Reply:
I am not sure we want to enforce it everywhere in Kibana with eslint as it would touch loads of area of the code base for sure. We just try to enforce it in our team by code review 😊